src/wp-includes/option.php

```diff
@@ -258,14 +258,24 @@ function get_option( $option, $default_value = false ) {
  *
  * @param array $options An array of option names to be loaded.
  */
-function wp_load_options( $options ) {
+function wp_prime_option_caches( $options ) {
 	$alloptions     = wp_load_alloptions();
-	$cached_options = wp_cache_get_multiple( $options, 'options' );
+	$cache_keys     = (array) $options;
+	$cache_keys[]   = 'notoptions';
+	$cached_options = wp_cache_get_multiple( $cache_keys, 'options' );
+
+	// Store notoptions, remove from cached options.
+	$notoptions = $cached_options['notoptions'];
+	unset( $cached_options['notoptions'] );
```

Comment:
Part of me thinks this implementation is a bit "too clever" and would be easier to follow by separating the two concerns, similarly to how `$alloptions` are retrieved separately from the actual `$options` provided.

I realize this implementation is technically a tiny bit more efficient as it avoids two separate `wp_cache_*()` retrieval calls, but I don't think that matters for performance in reality, so I personally would favor code simplicity here. That said, not a blocker either way.

Here's what I'm thinking:
```suggestion
	$notoptions     = wp_cache_get( 'notoptions', 'options' );
	$cached_options = wp_cache_get_multiple( $options, 'options' );
```